app/views/PaymentHistory.scala.html

```diff
@@ -78,14 +78,18 @@
         </tr>
         </thead>
         <tbody>
-        @for((payment, index) <- yearPayments.zipWithIndex) {
+        @for((payment, index) <- yearPayments.sortBy(x => x.date).reverse.zipWithIndex) {
             <tr class="govuk-table__row">
                 <td class="govuk-table__cell">@payment.date.toLongDate</td>
                 <td class="govuk-table__cell">
                     <a class="govuk-link" href="@payment.linkUrl">
                         @messages(payment.description)
                         <span class="govuk-visually-hidden">@payment.visuallyHiddenText @if(yearPayments.size > 1){ Item @(index + 1)}</span>
                     </a>
+                    <br>
+                    <a class="govuk-text">
+                        @messages("paymentHistory.taxYear", (payment.getTaxYearEndYear-1).toString, payment.getTaxYearEndYear.toString)
```

Comment:
We finally need to defined correct TaxYear type in order to remove usage of this pattern: line 91;
I would say its optional to apply this at this stage;

---------------------
PaymentHistory model should expose TaxYear property/model
TaxYear model will expose startYear: Int/String and endYear: Int/String

Here is example:
https://jira.tools.tax.service.gov.uk/browse/MISUV-6559